test/messaging.test.ts

```diff
@@ -8,6 +8,14 @@ import { Account } from "hardhat/types";
 import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
 import { getSelectorFromName } from "starknet/dist/utils/hash";
 
+const asset_symbol = 10703902247957299200;
```

Comment:
constants are usually written in `UPPER_CASE`, so `ASSET_SYMBOL` here. this comment also applies for lines below